metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -530,8 +531,11 @@ public void apply(final Metafix metafix, final Record record, final List<String>
                     if (c != null) {
                         c.accept(oldValue);
                     }
-
+                    elsif (defaultOption != null){
+                        return defaultOption;
+                    }
                     return defaultValue != null ? defaultValue : delete ? null : oldValue;
```

Comment:
```suggestion
                    return defaultOption != null ? defaultOption : defaultValue != null ? defaultValue : delete ? null : oldValue;
```

As [previously suggested](https://github.com/metafacture/metafacture-fix/issues/333#issuecomment-1801984070), this would be even better though (at the top of the method):

```java
final String defaultValue = options.getOrDefault(Maps.DEFAULT_MAP_KEY, map.get(Maps.DEFAULT_MAP_KEY));
```

It would allow you to specify the default value _per lookup call_ in addition to _per map_. The next step would then be to decide whether we want to keep supporting the Metamorph syntax (`__default`) or switch to the Catmandu syntax (`default`). Or even support both (which seems a little messy, though).